irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/exceptions/UsagePolicyPermissionException.java

```diff
@@ -27,15 +27,16 @@
 import org.eclipse.edc.policy.model.Policy;
 
 /**
- * Usage Policy Exception errors in the contract negotiation.
+ * Usage Policy Permission Exception errors in the contract negotiation.
  */
 @Getter
-public class UsagePolicyException extends EdcClientException {
+public class UsagePolicyPermissionException extends EdcClientException {
 
     private final transient Policy policy;
     private final String businessPartnerNumber;
 
-    public UsagePolicyException(final String itemId, final Policy policy, final String businessPartnerNumber) {
+    public UsagePolicyPermissionException(final String itemId, final Policy policy,
```

Comment:
Why Permission Expecption - The Policy does not comply. IMHO permission is not the right scope as policy itself does not comply which includes the mismatch of one or more permissions which are bundled inside a policy.